### PR TITLE
(docs) Add a defintion of 'slug' to Part 7 tutorial 

### LINKED ISSUE #21864: Add a defintion of 'slug' to Part 7 tutorial
### Refine the definition of “slug” in the Part 7 Tutorial

I think some extra information on what a ‘slug’ is could be a nice improvement given how often the term is used the tutorial. Users might not be 100% clear on what exactly ‘path’ is referring to.   

#### Current definition on https://www.gatsbyjs.org/tutorial/part-seven/
```
Creating slugs for pages 

Creating new pages has two steps:
1. Generate the “path” or “slug” for the page.
2. Create the page.
```
A possible suggestion:

```
Creating slugs for pages

A ‘slug’ is the unique identifying part of a web address, 
such as the `tutorial/part-seven` part of the page https://www.gatsbyjs.org/tutorial/part-seven/.

It is also referred to as the ‘path’ but we will use the term ‘slug’ during this tutorial.

 Creating new pages has two steps:
1. Generate the “path” or “slug” for the page.
2. Create the page.
```

### Motivation

Why should we document this and who will benefit from it?

Adding extra information to make the already __very__ beginner-friendly tutorial would be a good step in preventing people distracted by going to search for unfamiliar terms. I think the Gatsby tutorials already does a really good job of this, such as the explanation on GraphQL in Part 4. 

I'm happy to add further clarification or even admit this might not be necessary but I myself had to look up on what a `slug` meant after going through the tutorials. I'm hoping if there's others out there who also had to look up the term that maybe in the future this would prevent people from going to search for it and continue with the amazing tutorials! 😄 

### Draft the doc

- [ ] Write the doc, following the format listed in these resources:
  - [Overview on contributing to documentation](https://www.gatsbyjs.org/contributing/docs-contributions/)
  - [Docs Templates](https://www.gatsbyjs.org/contributing/docs-templates/)
  - [Example of a similar article]()
- [ ] Add the article to the [docs sidebar](https://github.com/gatsbyjs/gatsby/blob/master/www/src/data/sidebars/doc-links.yaml) under the [parent doc] section.

### Open a pull request

- [x] Open a pull request with your work including the words "closes #[this issue's number]" in the pull request description

### PR DESCRIPTION
**Pull request by mokyox**
## Related Issues

Closes #21864.

### Add/refine the definition of “slug” in the Part 7 Tutorial

I think some extra information on what a ‘slug’ is could be a nice improvement given how often the term is used the tutorial. Users might not be 100% clear on what exactly ‘path’ is referring to.   

#### Current definition on https://www.gatsbyjs.org/tutorial/part-seven/
```
Creating slugs for pages 

Creating new pages has two steps:
1. Generate the “path” or “slug” for the page.
2. Create the page.
```
A possible suggestion:

```
Creating slugs for pages

A ‘slug’ is the unique identifying part of a web address, 
such as the `tutorial/part-seven` part of the page https://www.gatsbyjs.org/tutorial/part-seven/.

It is also referred to as the ‘path’ but we will use the term ‘slug’ during this tutorial.

 Creating new pages has two steps:
1. Generate the “path” or “slug” for the page.
2. Create the page.
```

### Motivation

Why should we document this and who will benefit from it?

Adding extra information to make the already __very__ beginner-friendly tutorial would be a good step in preventing people distracted by going to search for unfamiliar terms. I think the Gatsby tutorials already does a really good job of this, such as the explanation on GraphQL in Part 4. 

I'm happy to add further clarification or even admit this might not be necessary but I myself had to look up on what a `slug` meant after going through the tutorials. I'm hoping if there's others out there who also had to look up the term that maybe in the future this would prevent people from going to search for it and continue with the amazing tutorials! 😄

### CODE REVIEW

File: docs/tutorial/part-seven/index.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ Let's get started.
 
 ## Creating slugs for pages
 
+A ‘slug’ is the unique identifying part of a web address,
+such as the `/tutorial/part-seven` part of the page https://www.gatsbyjs.org/tutorial/part-seven/.
```
Comment (laurieontech) [inline docs/tutorial/part-seven/index.md]:
```suggestion
such as the `/tutorial/part-seven` part of the page `https://www.gatsbyjs.org/tutorial/part-seven/`.
```

File: docs/tutorial/part-seven/index.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ Let's get started.
...
+
+It is also referred to as the ‘path’ but we will use the term ‘slug’ in this tutorial.
```
Comment (laurieontech) [inline docs/tutorial/part-seven/index.md]:
```suggestion
It is also referred to as the ‘path’ but this tutorial will use the term ‘slug’ for consistency.
```

**Comment by mokyox** [comment]
Thank you @laurieontech for the suggestions! They sound great - I've commited them into the PR now. 😄

**Comment by laurieontech** [comment]
This is really grat. Thanks so much for taking the time to add this context!

**Pull request merged**